### ISSUE TITLE
Allow listing a project as a derivative of a resource published elsewhere

### ISSUE THREAD

**Issue by bemoody** [MIT-LCP/physionet-build#2120]
It should be possible to list a project as being "derived from" a resource that was published on another site, by providing the DOI (or other URL?)

The parent resources should be shown in the "Parent Projects" info box (though "Projects" is maybe the wrong word to use), and should be listed in the DOI metadata (pull #2119) when the PhysioNet project is published.

**Comment by tompollard** [comment]
I agree, this would be good. Both the interface for adding parent projects (in the submission system) and the the box that displays parent projects (on the project description page) will need redesign.
